tests/snapshots/files__dead_loop_deletion-optimize-sequential.snap

```diff
@@ -14,18 +14,15 @@ expression: visualization.result
   v7_: int = id c3_;
   v8_: int = id c4_;
 .b9_:
```

Comment:
Same here. The old one does loop unrolling (step by 4) and the new one step by 1.

The old one also does not do the optimization (x+2)+2 = x + 4